### DISCUSSION TITLE
LaTeX Support in Markdown

### DISCUSSION THREAD

**Discussion by s0foo** [community/community#137022] [Mobile] [answered]
Hi,
Would it be possible to support LaTeX formatted math within Markdown on GitHub Mobile?
Cheers 🙂

**Comment by eliperkins** [comment, marked as answer]
Hey @s0foo, thanks for this feedback! I've added your +1 to our internal tracking issue for this which was [created for this original discussion post](https://github.com/orgs/community/discussions/18128). I'll work with the engineering team to figure out a path forward to get this addressed.

If you could add your upvote to the [original discussion post for LaTeX support on Mobile](https://github.com/orgs/community/discussions/18128), that'll help give us a better signal in one singular issue!
